src/USER-LD/examples/benzene_water/benzene_water.data

```diff
@@ -0,0 +1,1406 @@
+LAMMPS data file for 380 CG benzene, 1000 CG water particles
```

Comment:
Examples files should never be in the source tree. For a USER-MISC package, the corresponding folder would be `examples/USER/misc/<name>`, e.g. using `local_density` for `<name>`